.claude/skills/builder-smoke-test/scripts/scaffold.sh

```
@@ -0,0 +1,179 @@
+    echo "WORKOS_ORGANIZATION_ID=${WORKOS_ORG_ID}"
+    echo "WORKOS_REDIRECT_URI=http://localhost:4111/api/auth/callback"
+    echo "WORKOS_COOKIE_PASSWORD=${cookie_password}"
+  fi
+} > "${PROJECT_DIR}/.env"
+chmod 600 "${PROJECT_DIR}/.env"
+echo "✓ wrote .env"
+
+# 9. Install deps.
+if [ "${needs_install}" = "yes" ]; then
+  echo "→ pnpm install --ignore-workspace (this can take ~30-90s)"
+  pushd "${PROJECT_DIR}" >/dev/null
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)